src/mixins/field.js

```diff
@@ -11,6 +11,18 @@ import shared from './shared';
 export default {
   ...shared,
 
+  // Create a unique id for the field
+  getInitialState: function() {
+    return {
+      id:
+        this.props.field.id ||
+        Math.random()
```

Comment:
I'll state what is probably obvious: that for a _huge_ list of fields, this could mysteriously replicate. It's probably not a significant enough risk to worry about though.

Reply:
I feel like if you have a form with that many fields you should be tried for crimes against humanity, but I am definitely open to suggestions on this if there is a better way to generate an id